maquetado/scss/_tech.scss

```
@@ -0,0 +1,45 @@
+.tech {
+  align-items: center;
+  background-color: $green;
+  border-radius: 5px;
+  display: inline-flex;
+  flex-direction: column;
+  margin: $tech-decoration-offset 0 0 $tech-decoration-offset;
+  max-width: 240px;
+  padding: 2em;
+  position: relative;
+  width: 100%;
+
+  &:before {
```

```suggestion
  &::before {
```
Check this [article](https://css-tricks.com/to-double-colon-or-not-do-double-colon/)